### PR TITLE
Fix (1/2): [TOPI] conv2d schedule code (#3648 or #3717)

### PR DESCRIPTION
**Pull request by mingwayzhang**
* Updates the cuda configs for operator topi_nn_conv2d. [#3648](https://github.com/dmlc/tvm/issues/3648)

The change is done through the following commands:

`cat cuda_v0.04.log |grep topi_nn_conv2d| sed 's/tile_rx/tile_rxy/g; s/tile_ry/tile_rx/; s/tile_rxy/tile_ry/g' > cuda_v0.05.log`

followed by

`cat cuda_v0.04.log |grep -v topi_nn_conv2d >> cuda_v0.05.log`

Please review.

Thanks.

### CODE REVIEW

**Comment by mingwayzhang** [comment]
@merrymercy, could you review this one?

**Comment by mingwayzhang** [comment]
@cbalint13 @tmoreau89 

In order for #3717 to get in, this tophub change needs to be merged first.

Please review.

Thanks.

**Comment by tmoreau89** [comment]
Thank you for the fix

File: tophub/cuda_v0.05.log [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+{"i": ["cuda -model=titanx", "topi_nn_conv2d", [["TENSOR", [1, 640, 7, 7], "float32"], ["TENSOR", [128, 640, 1, 1], "float32"], [1, 1], [0, 0], [1, 1], "NCHW", "float32"], {}, ["conv2d", [1, 640, 7, 7, "float32"], [128, 640, 1, 1, "float32"], [1, 1], [0, 0], [1, 1], "NCHW", "float32"], {"i": 12506, "c": null, "e": [["tile_f", "sp", [8, 1, 16, 1]], ["tile_y", "sp", [7, 1, 1, 1]], ["tile_x", "sp", [1, 1, 7, 1]], ["tile_rc", "sp", [40, 16]], ["tile_rx", "sp", [1, 1]], ["tile_ry", "sp", [1, 1]], ["auto_unroll_max_step", "ot", 0], ["unroll_explicit", "ot", 0]], "t": "direct"}], "r": [[2.506711887295765e-05], 0, 2.169248342514038, 1535549800.977766], "v": 0.1}
+{"i": ["cuda -model=titanx", "topi_nn_conv2d", [["TENSOR", [1, 608, 7, 7], "float32"], ["TENSOR", [128, 608, 1, 1], "float32"], [1, 1], [0, 0], [1, 1], "NCHW", "float32"], {}, ["conv2d", [1, 608, 7, 7, "float32"], [128, 608, 1, 1, "float32"], [1, 1], [0, 0], [1, 1], "NCHW", "float32"], {"i": 54981, "c": null, "e": [["tile_f", "sp", [16, 1, 8, 1]], ["tile_y", "sp", [1, 1, 7, 1]], ["tile_x", "sp", [1, 1, 7, 1]], ["tile_rc", "sp", [38, 16]], ["tile_rx", "sp", [1, 1]], ["tile_ry", "sp", [1, 1]]
```
Comment (merrymercy) [inline tophub/cuda_v0.05.log:591]: This line should be put in the beginning of this file.

Comment (merrymercy) [reply tophub/cuda_v0.05.log:591]: #10

**Pull request merged**